scripts/changelogger.py

```diff
@@ -103,22 +135,34 @@ def create_changelog(version, change_dict):
     )
     args = parser.parse_args()
 
-    log_hist = git_log(args.ref1, args.ref2)
+    if args.ref1 and args.ref2:
+        log_hist = git_log(args.ref1, args.ref2)
+        headerVersion = args.ref2
+    elif not args.ref1 and not args.ref2:
+        latest_tags = git_latest_two_tags()
+        log_hist = git_log(*latest_tags)
+        headerVersion = latest_tags[-1]
+    else:
+        raise Exception(
+            "Either provide both --ref1 and --ref2 or neither (defaulting to latest two commits)"
```

Comment:
```suggestion
            "Either provide both --ref1 and --ref2 or neither (defaulting to latest two tagged commits)"
```

Reply:
Fixed the msg a bit differently to reflect what's actually used